app/views/provider_interface/interviews/index.html.erb

```diff
@@ -6,52 +6,16 @@
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
       <div class="app-interviews">
-        <%= govuk_link_to 'Set up interview', new_provider_interface_application_choice_interview_path(@application_choice), class: 'govuk-button govuk-button--secondary' %>
+        <% if @provider_can_respond %>
```

Comment:
Maybe update this to `@user_can_change_interview` so it's easier to understand how this variable is used here and in the `InterviewAndCourseSummaryComponent`.

Reply:
I've gone with `@provider_can_make_decisions` which actually describes what this variable means (rather than how it will be used inside a component)